### ISSUE TITLE
Automate the creation of new images at release time

### ISSUE THREAD

**Issue by laf** [librenms/packer-builds#1]
- [x] Hit the GitHub api for the latest release
  - [x] Build each image we want to produce
  - [x] Publish them automatically.
  - [ ] Have travis-ci from librenms/librenms kick start the process on tagged release.

**Comment by crazy-max** [comment]
> [`releases.sh`](https://github.com/librenms/packer-builds/blob/f89671cdf33240a733612ae2a577573499c9355f/release.sh#L45-L59)

@laf Why not [using Travis](https://docs.travis-ci.com/user/deployment/releases/) to upload assets ?

**Comment by laf** [comment]
@crazy-max Wasn't aware of that option. That would mean travis would have to build the actual images as well, would have to test that they support installing virtualbox and packer.

**Comment by crazy-max** [comment]
@laf Yep you can easily build your images on Travis and push the assets with an encrypted token. I can create a PR, I'm familiar with travis :)

**Comment by laf** [comment]
Go for it :)
